content/coronavirus_education_page.yml

```diff
@@ -18,13 +18,16 @@ content:
       - children in nursery, reception, year 1 and year 6 can return to school
       - schools remain closed for other year groups, except for children of key workers and vulnerable children
       - childminders can return to work
+      - schools will reopen to all age groups from September
   guidance_section:
     header: What you can do now
     list:
       - text: "Protective measures: keep children and staff safe when schools reopen"
         url: /government/publications/coronavirus-covid-19-implementing-protective-measures-in-education-and-childcare-settings/coronavirus-covid-19-implementing-protective-measures-in-education-and-childcare-settings
       - text: "School reopenings: prepare to reopen your school"
         url: /government/publications/actions-for-educational-and-childcare-settings-to-prepare-for-wider-opening-from-1-june-2020/actions-for-education-and-childcare-settings-to-prepare-for-wider-opening-from-1-june-2020
+       - text: Safe working in education, childcare and children’s social care 
```

Comment:
```suggestion
      - text: Safe working in education, childcare and children’s social care 
```

Reply:
Hi Issy - does this just need an indent?